CRM/Member/Tokens.php

```diff
@@ -76,4 +76,24 @@ public function getDependencies(): array {
     return ['fee' => 'membership_type_id'];
   }
 
+  /**
+   * Get any tokens with custom calculation.
+   *
+   * In this case 'fee' should be converted to{membership.membership_type_id.fee}
+   * but we don't have the formatting support to do that with no
+   * custom intervention yet.
+   */
+  protected function getBespokeTokens(): array {
+    return [
+      'fee' => [
+        'title' => ts('Membership Fee'),
```

Comment:
If we get to the point where the number formatting is settled I would deprecated this in favour of `{membership.membership_type_id.minimum_fee}`